<?xml version="1.0"?>
<launch>
    <arg name="markers_file_name" default="map"/>
    <arg name="marker_positions_file" default="$(find particle_filter)/maps/$(arg markers_file_name).yaml" />
    <arg name="cameras_file_name" default="cameras"/>
    <arg name="cameras_positions_file" default="$(find particle_filter)/maps/$(arg cameras_file_name).yaml" />
	
	<node name="particle_filter" pkg="particle_filter" type="particle_filterexec"  output="screen"  respawn="false">
	<rosparam command="load" file="$(arg marker_positions_file)"/>
        <rosparam command="load" file="$(arg cameras_positions_file)"/>
	<param name="IMAGE_WIDTH" value= "1812" />
	<param name="MARKER_HEIGHT" value = "0.545985" />
	<param name="MARKER_WIDTH" value = "0.3777" />
	<param name="NUM_CAM" value ="3" />
	</node>
	
        
</launch>
